client/my-sites/site-settings/form-general.jsx

```diff
@@ -353,6 +354,74 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	climateStrikeOption() {
+		const {
+			fields,
+			isRequestingSettings,
+			translate,
+			handleToggle,
+			handleSubmitForm,
+			isSavingSettings,
+		} = this.props;
+
+		const today = moment(),
+			lastDay = moment( { year: 2019, month: 9, day: 21 } );
+
+		if ( today.isAfter( lastDay, 'day' ) ) {
+			return null;
+		}
+
+		return (
+			<div>
+				<SettingsSectionHeader title={ translate( 'Climate Strike 2019' ) }>
+					<Button
+						compact={ true }
+						onClick={ handleSubmitForm }
+						primary={ true }
+						type="submit"
+						disabled={ isRequestingSettings || isSavingSettings }
+					>
+						{ isSavingSettings ? translate( 'Saving…' ) : translate( 'Save Settings' ) }
+					</Button>
+				</SettingsSectionHeader>
+				<Card>
+					<FormFieldset>
+						<CompactFormToggle
+							checked={ !! fields.climatestrike }
+							disabled={ isRequestingSettings }
+							onChange={ handleToggle( 'climatestrike' ) }
+						>
+							{ translate(
+								'This September, millions of us will walk out of our workplaces and homes to join ' +
+									'young climate strikers on the streets and {{climateStrikeLink}}demand an end to the age ' +
+									'of fossil fuels{{/climateStrikeLink}}. Show your solidarity with a {{digitalClimateStrikeLink}}digital ' +
+									'climate strike banner{{/digitalClimateStrikeLink}}. On September 20th the banner will expand to a dissmissable overlay.',
```

Comment:
Just wanted to note a typo that I spotted. :)

```suggestion
									'climate strike banner{{/digitalClimateStrikeLink}}. On September 20th the banner will expand to a dismissible overlay.',
```

Reply:
Thanks!